wbooks/src/App/NavBar/Controls/styles.scss

```diff
@@ -0,0 +1,26 @@
+.controls {
+  width: 160px;
+  height: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  .notifications {
+    content: url('../../../assets/notifications.svg');
+    width: 30px;
+    height: 30px;
+  }
+
+  .add-button {
+    content: url('../../../assets/add_book.svg');
+    width: 30px;
+    height: 30px;
```

Comment:
If you see that many elements share style you can add to them another class that has that style to avoid repetition, like:
```css
.control {
  width: 30px;
  height: 30px;
}
```